images/scripts/fedora-coreos.install

```diff
@@ -4,15 +4,15 @@ set -eux
 # install/upgrade RPMs that apply to Fedora CoreOS
 # Note: cockpit-selinux would be desirable, but needs setroubleshoot-server which isn't installed
 cd /var/tmp/build-results/
-rpm-ostree install cockpit-bridge-*.rpm cockpit-dashboard-*.rpm \
+rpm-ostree install cockpit-bridge-*.rpm \
```

Comment:
This causes the cockpit/fedora-coreos failures. I assume this PR needs to land in lockstep with the cockpit PR that removes it?

Reply:
Yeah, here I checked that all works as expected:
https://logs.cockpit-project.org/logs/pull-14905-20201130-193223-77c80724-fedora-coreos-bots-1423/log.html